elastic/kibana
# [Response Ops][Alerting] Add auth check to triggers_actions_ui config API

Resolves https://github.com/elastic/response-ops-team/issues/117

## Summary

* Added check to `internal/triggers_actions_ui/_config` API to check that user has access to at least one alerting rule type. If user does not, returns a forbidden error
* Renamed `listAlertTypes` to `listRuleTypes` to align with current terminology.

## To Verify

* Create a role & user that has no access to any alerting rule types
* Log in as user and navigate to `https://localhost:5601/internal/triggers_actions_ui/_config` and verify you get a forbidden response
* Update the role to include access to at least one alerting rule type
* Verify that user can now access the above API and there are no errors when creating a rule in the UI
